### PR TITLE
Peer feedback on style guide docs:  landing page, navigation, embed images, and multiple menu docs

### PR DESCRIPTION
**Pull request by rhetoric101**
I already received an initial review of these four documents from Austin in a previous PR, but this is a follow up PR since the initial PR has already been merged. So, this PR is for incorporating feedback from the last review:

Landing page template:
* I listed the different types of icons: Feather, third-party, and New Relic.
* Add var tags around INSERT placeholders.
* Clarified the bullet about "Insert a value for href...."

Left-pane navigation:
* In the first paragraph, I inserted absolute path pointing to nav directory of the development branch. I clarified that writers will probably update this nav file in their own branches. 
* In the second paragraph, I also used an absolute path to the GitHub repository, showing the location of the .yml file.
* I replaced the insights YAML example with the agent. Please confirm the link and the example snippet.

Embed images:
* I kept the link to the .js file since that seems to be the list of doc-specific icons (per Jerel).
* For the Gatsby theme icons, I switched the link to the Gatsby stuff imported into our docs site.

Docs in multiple menus
* I renamed the file and doc in the last PR from "dummy-docs" to "Include-a-doc-in-multiple-menus."
* I chopped out the arcane references to dummy docs. 
* I pushed up the nav YAML change for the new name.

### CODE REVIEW

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx [hunk 1]
```diff
@@ -96,10 +96,10 @@ You can choose from a variety of icons to include in your docs:
 * New Relic icons (prefixed with `nr-`)
 * Logos for third-party products (prefixed with `logo-`)
 
-Here are the two places you can look to see if we have the icon you need:
+Here are the two places you can look to see if we have the icon you need. At the moment, these locations have separate icons:
 
-* [Gatsby theme](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) (these are icons available across the developer and docs sites)
-* [Docs site icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104)
+* [Gatsby theme](https://github.com/newrelic/docs-website/tree/develop/src/%40newrelic/gatsby-theme-newrelic/icons) (these are icons available across the developer and docs sites)
+* [Docs site Feather icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104)
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/processes-procedures/embed-images.mdx:102]: Could we explain the difference between Gatsby and Feather icons?

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx [hunk 2]
```diff
@@ -125,8 +125,10 @@ If your icon appears inside running text, use the `<Icon>` component. Here are s
 
 ### Install new Feather icons 
 
-If you don't see the icon you want in either the [Gatsby theme](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) or in the [docs site icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104), here's how you can add a new icon to the Gatsby theme:
-
+If you don't see the icon you want in either the [Gatsby theme](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#available-icons) or in the [docs site Feather icons](https://github.com/newrelic/docs-website/blob/develop/src/@newrelic/gatsby-theme-newrelic/icons/feather.js#L100-L104), you can add a new icon to the Gatsby theme. Here's an example of adding a "database" icon:
+<Callout variant="tip">
+Instead of following the instructions below, you can also ask developers add the icon you want.
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/processes-procedures/embed-images.mdx:130]: "to add"

File: src/content/docs/style-guide/processes-procedures/update-left-navigation-pane.mdx [hunk 3]
```diff
@@ -4,29 +4,40 @@ redirects:
...
 ---
 
-Navigation for [docs.newrelic.com](https://docs.newrelic.com) is stored in `YAML` files located in the [src/nav](./src/nav) directory. Each top-level navigation should have its own configuration file. 
+Navigation for [docs.newrelic.com](https://docs.newrelic.com) is stored in `YAML` files located in the [https://github.com/newrelic/docs-website/tree/develop/src/nav](https://github.com/newrelic/docs-website/tree/develop/src/nav) directory. Each top-level navigation should have its own configuration file. You'll most likely edit a version of this file in your branch and merge it into the `develop` branch. 
 
 ## The configuration file
 
-Here is a snippet of the [src/nav/insights.yml](./src/nav/insights.yml) configuration. Note that the file has indentation that corresponds to the level of the navigation hierarchy. Be sure to imitate the same spacing when you make changes:
+As an example, here is a snippet of the [agents.yml](https://github.com/newrelic/docs-website/tree/develop/src/nav/agents.yml) navigation configuration. Note that the file has indentation that corresponds to the level of the navigation hierarchy. Be sure to imitate the same spacing when you make changes:
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/processes-procedures/update-left-navigation-pane.mdx:11]: "here's" (being very picky here)

**Comment by urbiz-nr** [comment]
Looks great, @rhetoric101 ! I've added a couple of comments, check them out because there's a "to" missing in a doc. Feel free to merge then!

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/processes-procedures/embed-images.mdx:102]: @urbiz-nr , I will add an explanation of the two groups.

File: src/content/docs/style-guide/processes-procedures/embed-images.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/processes-procedures/embed-images.mdx:130]: @urbiz-nr, good catch!

File: src/content/docs/style-guide/processes-procedures/update-left-navigation-pane.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/processes-procedures/update-left-navigation-pane.mdx:11]: @urbiz-nr, I changed that.

**Comment by mmfred** [comment]
@rhetoric101 - can you take a look to see what this conflict is and resolve it? Thanks!

**Pull request merged**